node/DEVELOPER.md

```diff
@@ -156,13 +156,16 @@ Development on the Node wrapper may involve changes in either the TypeScript or
 #### Running the linters
 
 1. TypeScript
+
     ```bash
-    # Run from the `node` folder
+    # Run from the root folder of the GLIDE repository
     npm install eslint-plugin-import@latest  @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-plugin-tsdoc eslint typescript eslint-plugin-import@latest eslint-config-prettier prettier
     npm i
+    cd node
     npx eslint . --max-warnings=0
     npx prettier --check .
     ```
+
```

Comment:
I recommend also to mention
```
npx prettier -w .
```
to auto-fix recommendations if possible

```
npx eslint --ignore-pattern ProtobufMessage.* .
```
to run from node dir, otherwise you'll have a lot of noise from protobuf-generated files

```
npx eslint --ignore-pattern ProtobufMessage.* --ignore-pattern 'build-ts/**' --fix .
```
to auto-fix